src/web/components/conditionalRoute/ConditionalRoute.jsx

```diff
@@ -0,0 +1,29 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import PropTypes from 'prop-types';
+
+import {Route, Redirect} from 'react-router-dom';
+import useCapabilities from 'web/hooks/useCapabilities';
+
+const ConditionalRoute = ({component: Component, feature, ...rest}) => {
+  const capabilities = useCapabilities();
+  const isEnabled = capabilities._featuresEnabled[feature];
```

Comment:
I think you might want to use the `featureEnabled` method instead.
```suggestion
  const isEnabled = capabilities.featureEnabled(feature);
```

Reply:
Yes, makes sense. Thanks!